napari/layers/_scalar_field/scalar_field.py

```diff
@@ -588,6 +590,106 @@ def _get_value(self, position):
 
         return value
 
+    def _get_value_ray(
+        self,
+        start_point: Optional[np.ndarray],
+        end_point: Optional[np.ndarray],
+        dims_displayed: list[int],
+    ) -> Optional[int]:
+        """Get the first non-background value encountered along a ray.
+
+        Parameters
+        ----------
+        start_point : np.ndarray
+            (n,) array containing the start point of the ray in data coordinates.
+        end_point : np.ndarray
+            (n,) array containing the end point of the ray in data coordinates.
+        dims_displayed : List[int]
+            The indices of the dimensions currently displayed in the viewer.
+
+        Returns
+        -------
+        value : Optional[int]
+            The first non-background value encountered along the ray. If none
+            was encountered or the viewer is in 2D mode, returns None.
```

Comment:
Note I changed the docstring compared to how it was: it said it would return the background value, instead of `None`.

When I worked to make sure that was the case, I actually broke some tests which expected `None`, so the docstring was wrong.